packages/main/cypress/specs/Tokenizer.cy.tsx

```diff
@@ -170,4 +170,40 @@ describe("Tokenizer - multi-line and Clear All", () => {
 			.find("[ui5-token]")
 			.should("have.length", 6);
 	});
+
+	it("handles overflow mechanism as expected", () => {
+		cy.mount(
+			<Tokenizer style={{ width: "100%", border: "1px solid red" }}>
```

Comment:
Thank you for the change @StefanDimitrov04 - small remark, you can strip this part "border: "1px solid red", it does not affect the test, less the markup, easier to read the test : )